### DISCUSSION TITLE
New homepage – Can't find interesting projects anymore, and the "Show less activity like this" button doesn't work

### DISCUSSION THREAD

**Discussion by ramnes** [community/community#67180] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hey team,

I'm trying to find my way on the new homepage, but it seems quite buggy.

I'm trying to see as many of the stars from the people I'm following as possible, but my feed is full of stars on one of my repositories, which is quite useless.

<details>
  <summary>Large image</summary>
  <img width="906" alt="image" src="https://github.com/community/community/assets/835072/71a70a9f-1523-4343-af4e-a595e6455212">
</details>

So I clicked the three dots, the "Show less activity like" button, and filled it that way:

<img width="468" alt="image" src="https://github.com/community/community/assets/835072/a1872119-8603-444f-abda-b090be51670d">

It's great that the option exists but... the result is still the same, nothing changed. My feed is still the same, full of stars on that repository.

This is what my feed used to look like, which is much better to keep track of new interesting projects:

<details>
  <summary>Large image</summary>
  <img width="1097" alt="image" src="https://github.com/community/community/assets/835072/83947724-f8a0-489b-8bbf-ef325ca1c3a3">
</details>

Would be very nice if you could restore "real time" stars.

**Comment by tolik518** [comment]
Yeah I agree, the new algorithm doesn't seem to work for me as well as the old one

**Reply by brendanzab** [reply]
We don’t even need a recommendation algorithm! Just an algorithm that shows us what the people we followed starred, and who they followed, in chronological order.

**Comment by ramnes** [comment]
Update 17/06/2024: still doesn't work
